Classes/View Controllers/BaseListViewController2.swift

```diff
@@ -13,15 +13,16 @@ protocol BaseListViewController2DataSource: class {
     func models(adapter: ListSwiftAdapter) -> [ListSwiftPair]
 }
 
-class BaseListViewController2: UIViewController,
+class BaseListViewController2<PageType: CustomStringConvertible>: UIViewController,
 ListSwiftAdapterDataSource,
-FeedDelegate {
+FeedDelegate,
+LoadMoreSectionController2Delegate {
 
     private let emptyErrorMessage: String
     public weak var dataSource: BaseListViewController2DataSource?
 
     public private(set) lazy var feed: Feed = { Feed(viewController: self, delegate: self) }()
```

Comment:
Why `public` here and for the `dataSource` 2 lines above? Isn't `internal` enough?

Reply:
Ya